.circleci/config.yml

```diff
@@ -46,7 +45,7 @@ jobs:
     environment:
       CC: gcc-9
       CXX: g++-9
-      CMAKE_EXTRA: -DCMAKE_BUILD_TYPE=Debug
+      CMAKE_EXTRA: -DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_FLAGS_DEBUG=
```

Comment:
why is this needed `DCMAKE_CXX_FLAGS_DEBUG=`?

Reply:
This set `CMAKE_CXX_FLAGS_DEBUG` to the empty string, instead of `-g`

`-DCMAKE_CXX_FLAGS_DEBUG=""` sets it to the string containing two double quotes ...